### ISSUE TITLE
CORS issues

### ISSUE THREAD

**Issue by nforro** [packit/dashboard#453]
For a while dashboard has been having issues with rejected API requests because of missing `Access-Control-Allow-Origin` header.

![error](https://github.com/user-attachments/assets/611180fa-061a-424a-99c7-b58103795bea)

I have to refresh multiple times (it varies, but usually more than 10 times) until an API request succeeds and dashboard becomes usable - once a requests succeeds, the problem goes away until I close the browser tab.

![requests](https://github.com/user-attachments/assets/f88b18b4-00ee-4a2a-b990-86387f6f42e6)

It happens across different OSes with different browser profiles, I use chromium-based browser but I'm quite sure it happened in Firefox as well, though I'm not able to reproduce it with Firefox now.

**Comment by Venefilyn** [comment]
@nforro this has to do with the API missing the header when returning 403. It's nothing to do with the dashboard. Why it's returning 403 I couldn't tell you. Maybe it's a specific pod that is unavailable and returning 403 Forbidden?

**Comment by nforro** [comment]
Makes sense, so there are two issues, one similar to https://github.com/packit/packit-service/issues/1883 and the other needs further investigation.
